common/compat.h

```diff
@@ -39,6 +39,7 @@
 
 #include <sys/types.h>
 #include <sys/stat.h>
+#include <sys/un.h>
```

Comment:
In the mingw CI:
```console
../common/compat.h:42:10: fatal error: sys/un.h: No such file or directory
   42 | #include <sys/un.h>
      |          ^~~~~~~~~~
```
I suppose a configure check is nevertheless needed :-(

Reply:
Ok, I'll update the PR